homeassistant/components/totalconnect/alarm_control_panel.py

```diff
@@ -235,6 +247,9 @@ def _arm_home_instant(self):
         """Arm home instant synchronous."""
         try:
             ArmingHelper(self._partition).arm_stay_instant()
+        except UsercodeInvalid:
+            _LOGGER.error("TotalConnect usercode is invalid. Did not arm home instant")
```

Comment:
Usually we raise `HomeAssistantError` so they get feedback in the UI.  You'll need to start the reauth before raising